From d482bcc8fca90afc95ac3cbe848a3c9d7ad81ddc Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Wed, 4 Nov 2020 23:37:44 -0800
Subject: [PATCH] autogen.sh: not generate configure

Only use autogen.sh to generate configure.ac.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>

refresh patch to 4.0
Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 autogen.sh | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 0aa7507..36e0423 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -33,10 +33,3 @@ if test -z $PKG_CONFIG; then
     echo "pkg-config is missing: please install it (apt-get install pkg-config) and try again"
     exit
 fi
-
-autoreconf -ivf
-
-echo "./configure $@"
-chmod +x configure
-./configure $@
-
-- 
2.25.1

